test/e2e/mcs_test.go

```diff
@@ -258,41 +258,6 @@ var _ = ginkgo.Describe("[MCS] Multi-Cluster Service testing", func() {
 			err := karmadaClient.PolicyV1alpha1().ClusterPropagationPolicies().Delete(context.TODO(), serviceImportPolicy.Name, metav1.DeleteOptions{})
 			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
 		})
```

Comment:
Can you add some comment to record why not wait `ServiceExport` and `ServiceImport` disappear on member clusters?

Reply:
OK, I will add it.